### ISSUE TITLE
Blur on 'launcher' layer produces odd results on launch with fuzzel

### ISSUE THREAD

**Issue by rehanzo** [WillPower3309/swayfx#179]
### Please read the following before submitting:
- Proprietary graphics drivers, including nvidia, are not supported. Please use the open source equivalents, such as nouveau, if you would like to use Sway.
- Please do NOT submit issues for information from the github wiki. The github wiki is community maintained and therefore may contain outdated information, scripts that don't work or obsolete workarounds.
  If you fix a script or find outdated information, don't hesitate to adjust the wiki page.

### Please fill out the following:
- **Swayfx Version:**
  - `sway -v`
  sway version 0.3

- **Debug Log:**
  - Run `sway -d 2> ~/sway.log` from a TTY and upload it to a pastebin, such as gist.github.com.
  - This will record information about sway's activity. Please try to keep the reproduction as brief as possible and exit sway.
  - Attach the **full** file, do not truncate it.
  https://gist.githubusercontent.com/RAR27/5addb1896ff966fe325eaad6a9861544/raw/5560b97cdcb6a7e9106852978f56642b668ec742/sway.log

- **Configuration File:**
  - Please try to produce with the default configuration.
  - If you cannot reproduce with the default configuration, please try to find the minimal configuration to reproduce.
  - Upload the config to a pastebin such as gist.github.com.
replicated on default config, only additions to the config I made was enabling blur for the launcher layer (`layer_effects "launcher" blur enable;`, adding a background, and adding a keybinding for fuzzel.

- **Description:**
  - The steps you took in plain English to reproduce the problem.

When opening up fuzzel, a large portion of the screen blurs for a second, before fuzzel opens up as it normally should. After it opens successfully, the blur is applied as expected, and the large blurred section goes away. I have a laptop connected to a monitor. When I open fuzzel on the monitor, the issue occurs. When I set fuzzels `dpi-aware` setting to off, it works as expected, which makes me think this is something on fuzzels side to do with dpi detection/setting. I will test with #159 soon though, as it seems this should be fixed with that PR.

**Comment by RAR27** [comment]
Alright, just tested with #159, works perfectly. Closing.
